.tekton/payload-tracker-go-sc-pull-request.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/RedHatInsights/payload-tracker-go?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
+      == "security-compliance"
+    pipelinesascode.tekton.dev/pipeline: https://github.com/RedHatInsights/konflux-pipelines/raw/v1.18.0/pipelines/docker-build-oci-ta.yaml
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: payload-tracker-sc
+    appstudio.openshift.io/component: payload-tracker-sc
+    pipelines.appstudio.openshift.io/type: build
+  name: payload-tracker-sc-on-pull-request
+  namespace: hcc-integrations-tenant
+spec:
+  params:
+  - name: git-url
+    value: '{{source_url}}'
+  - name: revision
+    value: '{{revision}}'
+  - name: output-image
+    value: quay.io/redhat-user-workloads/hcc-fr-tenant/hcc-integrations-tenant/payload-tracker-go-sc/payload-tracker-sc:on-pr-{{revision}}
```

Comment:
@ryemorris This PR should work with the current version of `konflux-release-data`. However, there's an inconsistency in `konflux-release-data` between the app name (payload-tracker-sc) and the Quay repo path (payload-tracker-go-sc). If you want to fix that in `konflux-release-data` (that's not mandatory), you'll need to edit the pipelines in this repo as well.